.github/policies/resourceManagement.yml

```
@@ -0,0 +1,145 @@
+id: 
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: 
+resource: repository
+disabled: false
+where: 
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description: 
+      frequencies:
+      - hourly:
+          hour: 3
+      filters:
+      - isIssue
+      - isOpen
+      - hasLabel:
+          label: doc-bug
+      - isNotLabeledWith:
+          label: okr-health
+      actions:
+      - addLabel:
+          label: okr-health
+    eventResponderTasks:
+    - if:
+      - payloadType: Pull_Request
+      then:
+      - inPrLabel:
+          label: in-pr
+      description: Add in-pr label to issues
+    - if:
+      - payloadType: Pull_Request
+      - isAction:
+          action: Opened
+      - and:
+        - not:
+            activitySenderHasPermission:
+              permission: Admin
+        - not:
+            activitySenderHasPermission:
+              permission: Admin
+        - not:
+            activitySenderHasPermission:
+              permission: Write
+        - not:
+            isActivitySender:
+              user: github-actions[bot]
+              issueAuthor: False
+        - not:
+            isActivitySender:
+              user: github-actions
+              issueAuthor: False
+        - not:
+            isActivitySender:
+              user: azure-sdk
+              issueAuthor: False
+        - not:
+            isActivitySender:
+              user: dependabot
+              issueAuthor: False
+      then:
+      - addLabel:
+          label: community-contribution
+      description: Label community PRs
+    - if:
+      - payloadType: Pull_Request
+      - isActivitySender:
+          user: azure-sdk
+          issueAuthor: False
+      then:
+      - approvePullRequest:
+          comment: "Approved, we'll :shipit: when all status checks pass."
+      - addLabel:
+          label: azure-sdk-shipit
+      description: Label PRs from the Azure SDK bot
+    - if:
+      - payloadType: Pull_Request
+      - hasLabel:
+          label: azure-sdk-shipit
+      then:
+      - enableAutoMerge:
+          mergeMethod: Squash
+      description: Auto-merge Azure SDK PRs with green builds
+    - if:
+      - payloadType: Pull_Request
+      - labelRemoved:
+          label: azure-sdk-shipit
+      then:
+      - disableAutoMerge
+      description: Auto-merge Azure SDK PRs with green builds

```

I don't see a corresponding rule in the previous configuration, and I don't know if this is desired. @BillWagner / @gewarren -- this new rule would disable auto-merge for Azure SDK PRs if the https://github.com/dotnet/docs/labels/azure-sdk-shipit label gets removed. Would you like to gain that rule?

```suggestion
    - if:
      - payloadType: Pull_Request
      - labelRemoved:
          label: azure-sdk-shipit
      then:
      - disableAutoMerge
      description: Disable auto-merge of Azure SDK PRs if the label gets removed

```

---

Leaving this as-is. We can augment the automation after migration if needed.